polygeist/tools/cgeist/Test/Verification/sycl/global_string.cpp

```diff
@@ -0,0 +1,19 @@
+//RUN: clang++ -fsycl -fsycl-device-only -emit-mlir %s -S -o - 2>&1| FileCheck %s --check-prefix=MLIR
```

Comment:
```suggestion
// RUN: clang++ -fsycl -fsycl-device-only -emit-mlir %s -S -o - 2>&1| FileCheck %s --check-prefix=MLIR
```
Can you please format the test case?

Reply:
Can you please format the test case in general?

Reply:
Done